src/app/home/page.tsx

```diff
@@ -42,49 +44,57 @@ export default function Index() {
     if (localStorage.getItem('username')) {
       getMyEvents(localStorage.getItem('username') as UUID)
     }
+    setUpcomingEvents(
+      JSON.parse(
+        localStorage.getItem('FindingATimeRecentlyViewed') as string,
+      ) as Event[],
+    )
   }, [])
 
   return (
     <div className="container mx-auto p-4">
-      {' '}
-      {/*main container with max width auto center and padding 4*/}
       <div className="flex">
-        {' '}
-        {/*flex container to arrange child elements in a row*/}
-        <div className="w-2/3 p-4">
-          {' '}
-          {/*container for left side 2/3 of the width and has padding 4*/}
+        <div className="w-3/4 p-4">
           <div className="mb-4 flex items-center justify-start">
-            {' '}
-            {/*flex container to align heading and button horizontally to have them closer together*/}
-            <h1 className="text-2xl font-black font-extrabold">My Events</h1>{' '}
-            {/*heading for MY Events*/}
+            <h1 className="text-2xl font-black font-extrabold">My Events</h1>
             <Link href="/create-event">
-              <button className="btn btn-primary ml-6">+ New Event</button>{' '}
+              <button className="btn btn-primary ml-6">+ New Event</button>
             </Link>
-            {/*button to add new event with primary styling and margin left 6*/}
           </div>
           <div className="grid grid-cols-3 gap-4">
-            {' '}
-            {/*grid container with 2 columns and gap between the items*/}
-            {events.length > 0 &&
-              events.map((event) => (
+            {myEvents.length > 0 &&
```

Comment:
anything to show when empty?

Reply:
Now displays a message to tell users "No events found. Press "New Event" to create one!".